### PR TITLE
cleaned up - Ellow again, i cleaned up and applied your suggestions

### PR DESCRIPTION
**Pull request by GentleHome**
I will do the change profile after my shift, thank

### CODE REVIEW

File: repo/views.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 from django.http import FileResponse
 from django.shortcuts import render, redirect
-from django.contrib.auth import login, logout, authenticate
+from django.contrib.auth import login, logout, authenticate, update_session_auth_hash
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
-from django.contrib.auth.forms import AuthenticationForm
-from repo.forms import ChangeEmailForm, FileUploadForm, FileUploadToFolderForm, RegistrationForm, FolderRenameForm, FileRenameForm, ChangePassWordForm, ChangeUsernameForm, ChangeEmailForm
+from django.contrib.auth.forms import AuthenticationForm, PasswordChangeForm
+from repo.forms import *
```
Comment (YourAlly) [inline repo/views.py:7]: Thank you for this

File: repo/templates/repo/profile.html [hunk 2]
```diff
@@ -3,32 +3,14 @@
...
-
-<div class="container py-5">
-    <h1>{{action2}}</h1>
-    <form method="POST" action="{% url 'change-email' %}">
-        {% csrf_token %}
-        {{ form2|crispy }}
-        <button class="btn btn-primary" type="submit">Submit</button>
-    </form>
-</div>
+<a href="{% url 'change-password' %}">
```
Comment (YourAlly) [inline repo/templates/repo/profile.html:6]: Edit profile forms will be on separate pages now?

**Review by YourAlly — APPROVED** [review]
I'll approve, there are conflicts tho

still a poggers pr

File: repo/templates/repo/profile.html (hunk 2 above)
Comment (GentleHome) [reply repo/templates/repo/profile.html:6]: I used your forms.html

**Pull request merged**